Documentation/devel/release.md

```diff
@@ -46,15 +46,21 @@ The rkt version is [hardcoded in the repository](https://github.com/coreos/rkt/b
 - File a PR and get a review from another [MAINTAINER](https://github.com/coreos/rkt/blob/master/MAINTAINERS).
   This is useful to a) sanity check the diff, and b) be very explicit/public that a release is happening
 - Ensure the CI on the release PR is green!
+- Merge the PR
 
-After merging and going back to master branch, we check out the release version and tag it:
+Check out the release version and build it!
```

Comment:
While you're here, maybe you could clarify exactly what it means to "check out the release version"? (i.e. the exact commit where the version was set?)